test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -503,7 +505,7 @@ function compute_gradient_flux!(
     K_h = en_dif.K_m / Pr_t
     Diss₀ = turbconv.mix_len.c_d * sqrt(tke_en) / en_dif.l_mix
 
-    en_dif.shear_prod = ρa₀ * en_dif.K_m * gm_dif.S² # tke Shear source
+    en_dif.shear_prod = ρa₀ * en_dif.K_m * Shear² # tke Shear source
```

Comment:
Can we trust these `en_dif` diagnostics?

Reply:
Probably not. Let's address that in a subsequent PR as these are behavioral changes and removing these diagnostics will not be.